### PR TITLE
Send edited entries

### PR DESCRIPTION
**Pull request by elpaso**

### CODE REVIEW

**Review by Xpirix — APPROVED** [review]
Looks good to me

File: qgisfeedproject/qgisfeed/models.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ class QgisFeedEntry(models.Model):
 
     # Dates
     publish_from = models.DateTimeField(_('Publication start'), auto_now=False, auto_now_add=False, blank=True, null=True, db_index=True)
-    publish_to = models.DateTimeField(_('Publication end'), auto_now=False, auto_now_add=False, blank=True, null=True, db_index=True)
+    publish_to = models.DateTimeField(_('Publication end'), auto_now=False, auto_now_add=False, blank=True, null=True, db_index=True, help_text=_('The entry will be hidden to users who after this date. You can set this in the past to temove the entry from the users listing.'))
```
Comment (elpaso) [inline qgisfeedproject/qgisfeed/models.py:97]:
```suggestion
    publish_to = models.DateTimeField(_('Publication end'), auto_now=False, auto_now_add=False, blank=True, null=True, db_index=True, help_text=_('The entry will be hidden to users after this date. You can set this in the past to temove the entry from the users listing.'))
```

**Comment by elpaso** [comment]
@Xpirix please let me know when it's deployed

**Comment by Xpirix** [comment]
@elpaso Sure. We will plan the deployment with @dimasciput and will let you know when it's deployed.

**Comment by Xpirix** [comment]
@elpaso , These changes are now deployed.

**Pull request merged**